extension-reference/mobile/analytics/integration.md

```diff
@@ -0,0 +1,35 @@
+```
+---
+description: >-
+  This section helps app developers understand how other modules can be integrated with Analytics.
+---
+```
+
+# Analytics Integration
+
+There are several Adobe provided extensions that can be integrated with Analytics. This document will cover the use cases on integrating with Analytics. The following integrations are all optional, except for Identity which is required by Analytics.
+
+## Acquisition
+
+**Application First Install:** The first time a user installs and opens you app the Analytics module is responsible for sending an install hit containing the referrer information from Acquisition if available.
+
+## Lifecycle
+
+**Application First Install:** The first time a user installs and opens you app the Analytics module is responsible for sending an install hit containing the install, launch, and crash information from Lifecycle if available.
```

Comment:
We can have one section for all the lifecycle use cases. Also we should mention that the lifecycle information is sent in each analytics hit.

> In order to track the application lifecycle details, you need to properly configure the lifecycle data collection. This will enable you to track information about application installs or upgrades, application crash events, application launch details (for example: number of launches, session length). In addition to that, every analytics hit will contain the default lifecycle metrics, which include information about the device being used, operating system, device resolution etc. For more details please check the lifecycle section \<link> and the lifecycle metrics details description \<link>.